### PR TITLE
TGIF 102

### PR DESCRIPTION
**Pull request by gr2m**
[Preview](https://github.com/hoodiehq/hood.ie/blob/tgif-102/_posts/2016-10-21-sweet-break-before-break-102.md)

### CODE REVIEW

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+---
+
+Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday! This Friday’s little bit of knowledge, fascination, and fun is curated for you by [Rob](https://twitter.com/gibbiv) :)
+
+## Editors’ Note: A short hiatus
+
+Last week I hopped on a call with [Cecily](https://twitter.com/skeskali), [Gregor](https://twitter.com/gr2m), and [Jenn](https://twitter.com/jennwrites) to discuss the future of TGIF, Hoodie’s lazy Friday afternoon read started by [Lena](https://twitter.com/lrnrd) in [August 2013](http://hood.ie/blog/tgif-hoodies-top-links-of-the-week-1.html). After an hour-long chat with these amazing peeps we decided to take a step back and rethink how Hoodie’s weekly newsletter is created, delivered and promoted.
+
+Some of the things we discussed were TGIF having it’s own platform (separate from the blog), opening each issue up to multiple contributors (one contributor per section), making TGIF more accessible (with an email subscription option), and reaching out to publishers of content we share (to spread awareness).
```
Comment (machikoyasuda) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: `its` not `it's (It is)`

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+I’m grateful to be part of this project and look forward to making TGIF one of the most anticipated newsletters in open source, intellect and fun! Now for what you may have missed over the past 2 weeks...
+
+## Open Source
+
+[Obama’s Facebook Messenger bot is open source](https://www.whitehouse.gov/blog/2016/10/13/removing-barriers-constituent-conversations) (whitehouse.gov)
+
+Obama reads ten letters a day, but he can’t read em’ all, so he decided to scale responses with the Obama bot: a handy Drupal [module](https://github.com/WhiteHouse/fb_messenger_bot) that you (or any government) can now use to build a chat bot on Facebook's Messenger platform.
+
+[FreeBSD stable/11 branch released](https://www.freebsd.org/releases/11.0R/announce.html) (freebsd.org)
```
Comment (machikoyasuda) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: Let's always spell out abbreviations on the first reference. This saves people a Google search (or, for me, a DuckDuckGo search) to learn something new. BSD, I just learned right now, stands for `Berkeley Software Distribution`. So something like: `FreeBSD (Berkeley Software Distribution) stable/11 branch released`

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+The OS that powers [many](https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/nutshell.html%23introduction-nutshell-users) successful projects on the web is born again, with security improvements and better hardware support.
```
Comment (machikoyasuda) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: `operating system (OS)`

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+A swipe file-like post with examples of UX design that’s so brilliant it goes unnoticed.
+
+[A blog from Esty’s design team](https://medium.com/etsy-design) (https://medium.com/etsy-design)
+
+The design team at Etsy has a new blog that’s worth checking out.
+
+## Web and Development
+
+[Open guide for AWS](https://github.com/open-guides/og-aws) (github.com)
+
```
Comment (machikoyasuda) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: Amazon Web Services (AWS)

**Comment by machikoyasuda** [comment]
Thank you for crafting this newsletter!

I noticed we have a lot more links than usual. One of the great privledges of being a writer and editor is choosing the best of the best. Feel free to take any links _out_ if you don't 100% love them. 

I also noticed there are a lot of references to Google. Maybe we can choose the one or two _best_ news coming out from Google instead, and mix it up with smaller companies and groups that people may not know about. 

The other theme is to make sure to spell out all abbreviated names, titles on first reference. That's all! TGIF!!

**Review by machikoyasuda — CHANGES_REQUESTED** [review]
1. Spell out all abbreviations (BSD, OS for example)
2. Trim down the number of links to your favorite ones.

**Comment by gr2m** [comment]
@igotpassion can you review it this weekend? No worries that it gets published late, still great content :)

File: _posts/2016-10-21-sweet-break-before-break-102.md (hunk 2 above)
Comment (igotpassion) [reply _posts/2016-10-21-sweet-break-before-break-102.md]: Great point.

**Review by igotpassion — APPROVED** [review]

**Comment by igotpassion** [comment]
@gr2m The requests by @machikoyasuda are on point. Struggling without my comp (it finally burnt out from the water spill). I should be online on my friend's iPad for an hour before I hit the road again (traveling in Costa Rica). Let me know if I can do anything else. Thanks so much to you both for helping with this!

**Comment by gr2m** [comment]
hey sorry I hope I would have time to finish this up over the weekend but didn’t :( let's make it ready for next Friday, okay? Very sorry for that

**Comment by gr2m** [comment]
@igotpassion can you incorporate Machiko’s feedback? Maybe put in some more recent links that came up over the week? Sorry again that I didn’t manage to publish this last Friday :(

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+## Editors’ Note: A short hiatus
+
+Last week I hopped on a call with [Cecily](https://twitter.com/skeskali), [Gregor](https://twitter.com/gr2m), and [Jenn](https://twitter.com/jennwrites) to discuss the future of TGIF, Hoodie’s lazy Friday afternoon read started by [Lena](https://twitter.com/lrnrd) in [August 2013](http://hood.ie/blog/tgif-hoodies-top-links-of-the-week-1.html). After an hour-long chat with these amazing peeps we decided to take a step back and rethink how Hoodie’s weekly newsletter is created, delivered and promoted.
+
+Some of the things we discussed were TGIF having it’s own platform (separate from the blog), opening each issue up to multiple contributors (one contributor per section), making TGIF more accessible (with an email subscription option), and reaching out to publishers of content we share (to spread awareness).
+
+I’m grateful to be part of this project and look forward to making TGIF one of the most anticipated newsletters in open source, intellect and fun! Now for what you may have missed over the past 2 weeks...
```
Comment (jennwrites) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: “in open source, intellect, and fun” might not translate well to ESL readers, I think you could be more concise here

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+[Open source and business](https://changelog.com/rfc/8) (changelog.com)
+
+David Cramer (CEO of Sentry) and Isaac Schlueter (CEO of npm) discuss different company approaches to developing open source projects.
+
+[Google discovers key to good teamwork](http://qz.com/625870/after-years-of-intensive-analysis-google-discovers-the-key-to-good-teamwork-is-being-nice/) (qz.com)
+
+Google’s data-driven approach ended up highlighting what leaders in the business world have known for a while: just be nice.
+
+## Events
```
Comment (jennwrites) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: Quite a few of the upcoming events [mentioned in the last TGIF](http://hood.ie/blog/hacktober-thumbs-high-level-dumplings-tgif-101.html) are missing here.

File: _posts/2016-10-21-sweet-break-before-break-102.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+## We <3
+
+[Janice Fraser’s talk about women in leadership](https://www.youtube.com/watch?v%3DSDIV8XV6Qrg)
+
+[New Yorker article about learning to become resilient](http://www.newyorker.com/science/maria-konnikova/the-secret-formula-for-resilience)
+
+[Tim Wu’s latest book The Attention Merchants](http://www.timwu.org/AttentionMerchants.html)
+
+[Github’s community guidelines](https://www.google.com/url?sa%3Dt%26rct%3Dj%26q%3D%26esrc%3Ds%26source%3Dweb%26cd%3D1%26cad%3Drja%26uact%3D8%26ved%3D0ahUKEwj5--qrv-zPAhXJjz4KHT_0CAsQFggeMAA%26url%3Dhttps%253A%252F%252Fgithub.com%252Fblog%252F2267-introducing-github-community-guidelines%26usg%3DAFQjCNHmxcYBP9G9WhSUirqx3L7nWOslHg%26sig2%3D0p0Z9rhqUBTu6uSb07WpRw)
```
Comment (jennwrites) [inline _posts/2016-10-21-sweet-break-before-break-102.md]: s/Github/GitHub

**Comment by igotpassion** [comment]
Thanks for the feedback y'all and apologies for the delay. I'll update this with some newer links, incorporate changes, and have ready for this Friday 👍

**Review by igotpassion — APPROVED** [review]

**Review by igotpassion — APPROVED** [review]

**Comment by gr2m** [comment]
I’ll be out tomorrow, @skeskali @renrutnnej @machikoyasuda can you please help get the TGIF out tomorrow?

**Comment by machikoyasuda** [comment]
ill be out tomorrow as well!!

**Comment by skeskali** [comment]
I think I can get it done, but I haven't published directly to the blog before. Is it just a matter of merging the request, or are there other steps?

**Comment by igotpassion** [comment]
@skeskali I'm making final changes now (hopefully I'm doing it right as this is my first time using GitHub). I will update you when ready.

**Review by igotpassion — APPROVED** [review]

**Comment by gr2m** [comment]
I’m sorry this got pushed off another week. I’m done with organizing Offline Camp and will be free this Friday, so can definitely help to get it out this time

**Review by machikoyasuda — APPROVED** [review]
👍🏽

**Pull request merged**